src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx

```diff
@@ -39,17 +40,17 @@ Before CloudWatch metric streams, the only solution for AWS monitoring partners
       <td>
         **All CloudWatch metrics from all AWS services and custom namespaces are available** in New Relic at once, without needing a specific integration to be built or updated.
 
-        There's one exception: metrics that are made available to CloudWatch with more than 2 hours delay are not included in the stream.
+        There's one exception: When metrics are available to AWS CloudWatch metric stream with more than 2 hours delay, these metrics aren't included in the stream.
```

Comment:
metrics are made available in `CloudWatch` service, not metric stream.
Probably correct way would be `...When metrics are available to AWS CloudWatch with more than...`